plugins/mission/mission_item_impl.h

```diff
@@ -48,6 +50,7 @@ class MissionItemImpl {
     float _relative_altitude_m = NAN;
     float _speed_m_s = NAN;
     bool _fly_through = false;
+    float _acceptance_radius_m = 0.0;
```

Comment:
I would like to have this consistent with the other properties. So I would vote to set the default to `NAN`.

Reply:
makes sense